CRM/Core/SelectValues.php

```diff
@@ -1203,4 +1204,25 @@ public static function relationshipOrientation() {
     ];
   }
 
+  /**
+   * @return array
+   */
+  public static function timezone() {
+    static $tzlist = NULL;
```

Comment:
Hi, not sure if this is still WIP but I have some nitpicky comments completely unrelated to the functionality. Feel free to ignore if still working on things.

Use of `static` as a cache is discouraged - it causes problems in unit tests and scripts because it doesn't reset in between - use Civi::$statics (see https://civicrm.org/blog/totten/the-static-is-dead-long-live-the-static or other examples in core).

Reply:
Thanks for reviewing!
Yes completely forgot about the Civi::$statics pattern for this one, will change.